tests/integration/CoreApiIntegrationTest.php

```diff
@@ -112,4 +112,111 @@ public function testChargeGopay()
+    {
+        try {
+            $this->charge_response = CoreApi::getPaymentAccount("dummy");
+        } catch (\Exception $e) {
+            $this->assertContains("Midtrans API is returning API error.", $e->getMessage());
+        }
+    }
+
+    public function testUnlinkPaymentAccount()
+    {
+        try {
+            $this->charge_response = CoreApi::unlinkPaymentAccount("dummy");
+        } catch (\Exception $e) {
+            $this->assertContains("Account doesn't exist.", $e->getMessage());
+        }
+    }
+
+    public function testCreateSubscription()
+    {
+        $param = array(
+            "name" => "Monthly_2021",
+            "amount" => "10000",
+            "currency" => "IDR",
+            "payment_type" => "credit_card",
+            "token" => "dummy",
+            "schedule" => array(
+                "interval" => 1,
+                "interval_unit" => "month",
+                "max_interval" => "12",
+                "start_time" => "2022-08-17 10:00:01 +0700"
+            ),
+            "metadata" => array(
+                "description" => "Recurring payment for user a"
+            ),
+            "customer_details" => array(
+                "first_name" => "John",
+                "last_name" => "Doe",
+                "email" => "johndoe@gmail.com",
+                "phone_number" => "+628987654321"
+            )
+        );
+        $this->charge_response = CoreApi::createSubscription($param);
+        $this->assertEquals('active', $this->charge_response->status);
+        $this->subscriptionId = $this->charge_response->id;
+    }
+
+    public function testGetSubscription()
+    {
+        try {
+            $this->charge_response = CoreApi::getSubscription("dummy");
+        } catch (\Exception $e) {
+            $this->assertContains("Subscription doesn't exist.", $e->getMessage());
+        }
+    }
+
+    public function testDisableSubscription()
+    {
+        try {
+            $this->charge_response = CoreApi::disableSubscription("dummy");
+        } catch (\Exception $e) {
+            $this->assertContains("Subscription doesn't exist.", $e->getMessage());
+        }
+    }
+
+    public function testEnableSubscription()
+    {
+        try {
+            $this->charge_response = CoreApi::enableSubscription("dummy");
+        } catch (\Exception $e) {
+            $this->assertContains("Subscription doesn't exist.", $e->getMessage());
+        }
+    }
+
+    public function testUpdateSubscription()
+    {
+        $param = array(
+            "name" => "Monthly_2021",
+            "amount" => "25000",
+            "currency" => "IDR",
+            "token" => "dummy",
+            "schedule" => array(
+                "interval" => 1
+            )
+        );
+
+        try {
+            $this->charge_response = CoreApi::updateSubscription("dummy", $param);
+        } catch (\Exception $e) {
+            $this->assertContains("Subscription doesn't exist.", $e->getMessage());
```

Comment:
Please add more test with positive-scenario (result in success 2xx) , for reference:
- https://github.com/Midtrans/midtrans-go/blob/main/coreapi/clienttokenization_test.go
- https://github.com/Midtrans/midtrans-go/blob/main/coreapi/clientsubscription_test.go

Because if your test only cover negative scenario, in the future if you change some implementation and run the test, it will not be able to help you spot some failure on positive scenario. And it is more important to test for the positive scenario, as it the main purpose of this library is to help make a success API request.

Thanks